core/server/services/members/api.js

```diff
@@ -79,6 +71,18 @@ let mailer;
 
 const membersConfig = config.get('members');
 
+function validateAudience({audience, origin}) {
+    if (audience === origin) {
```

Comment:
ATM a site can always equets a members token for their site - this enables SSO out of the box